aws/internal/keyvaluetags/generators/createtags/main.go

```diff
@@ -133,25 +134,27 @@ func isResourceTimeoutError(err error) bool {
 // {{ . | Title }}CreateTags creates {{ . }} service tags for new resources.
```

Comment:
`isAWSErrCode()` and `isAWSErrCodeContains()` above can also be removed now and replaced by functions in `github.com/hashicorp/aws-sdk-go-base/tfawserr`.